packages/enzyme-test-suite/test/_helpers/getLoadedLazyComponent.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { fakeDynamicImport } from 'enzyme-adapter-utils/src/Utils';
+import { is } from './version';
+
+function fakeSyncThenable(result) {
+  return {
+    then(resolve) {
+      return resolve({ default: result });
+    },
+  };
+}
+
+export default function getLoadedLazyComponent(wrappedComponent) {
+  if (is('>= 16.8')) {
+    return React.lazy(() => fakeSyncThenable(wrappedComponent));
```

Comment:
i missed this in the refactoring; `lazy` should be imported from `react-compat` here.